src/applications/vaos/tests/e2e/va-appointment.cypress.spec.js

```diff
@@ -214,213 +214,3 @@ describe('VAOS direct schedule flow', () => {
     newApptTests.confirmationPageTest(additionalInfo);
   });
 });
-
-describe('VAOS direct schedule flow: 2-step facility selection', () => {
-  it('should submit form', () => {
-    initAppointmentListMock();
-    initVAAppointmentMock({ facilityPageV2Enabled: false });
```

Comment:
deleting tests that's relying on flat facility toggle flag that's no longer used